app/src/main/java/com/github/multimatum_team/multimatum/DependenciesProvider.kt

```diff
@@ -32,6 +35,14 @@ object DependenciesProvider {
         applicationContext.getSystemService(SENSOR_SERVICE) as SensorManager
 }
 
+@Module
+@InstallIn(SingletonComponent::class)
+object ClockModule {
```

Comment:
Just a question: why you separate it in another module ? It is to have a better code ?

Reply:
If I put the clock service in our DependenciesProvider module, I'd have to mock the sensor and shared preferences as well in the tests, even if the tested code doesn't use any of those. The way I understand it, the provided dependencies in a module should be coupled together (e.g. if I have another dependency related to repositories, I'd put it in the repository module).